src/providers/twitch/TwitchChannel.cpp

```diff
@@ -909,22 +909,25 @@ void TwitchChannel::refreshCheerEmotes()
                     // We will continue to do so for now since we haven't had to
                     // solve that anywhere else
 
+                    // Combine the prefix (e.g. BibleThump) with the tier (1, 100 etc.)
+                    auto emoteTooltip =
+                        set.prefix + tier.id + "<br/>Twitch Cheer Emote";
```

Comment:
small nitpick, just to keep consistency with all the other `<br>`'s in the codebase
```suggestion
                        set.prefix + tier.id + "<br>Twitch Cheer Emote";
```

Reply:
Resolved.